admin/src/util/reportRequestError.js

```diff
@@ -9,12 +9,16 @@ const REST_NO_ROUTE_ERROR = __( 'Oh no! Your web browser could not reach your Wo
 const REST_COOKIE_INVALID_NONCE_ERROR = __( 'It looks like your web browser session expired. Try logging out and log back in to WordPress admin.', 'font-awesome' )
 const OK_ERROR_PREAMBLE = __( 'The last request was successful, but it also returned the following error(s), which might be helpful for troubleshooting.', 'font-awesome' )
 const ONE_OF_MANY_ERRORS_GROUP_LABEL = __( 'Error', 'font-awesome' )
+const FALSE_POSITIVE_MESSAGE = __( 'The last request contained errors, though your WordPress server reported it as a success. This usually means there\'s a problem with your theme or one of your other plugins emitting output that is causing problems.', 'font-awesome' )
```

Comment:
@frrrances The next few error message are new copy in this PR. Would you like to revise them? None of these would end up in UI, only in the JavaScript console when errors are reported.

Reply:
These seem good to me 👍